Assets/MixedRealityToolkit.SDK/Experimental/Features/Utilities/TapToPlace/TapToPlace.cs

```diff
@@ -0,0 +1,467 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Physics;
+using Microsoft.MixedReality.Toolkit.SpatialAwareness;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using Microsoft.MixedReality.Toolkit.Utilities.Solvers;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.Utilities
+{
+    /// <summary>
+    /// Tap to place is a far interaction component used to place objects on a surface.
+    /// </summary>
+    public class TapToPlace : Solver, IMixedRealityPointerHandler
+    {
+        [Experimental]
```

Comment:
Just to check, [Experimental] is being used on the very first thing to show the experimental warning right?

Actually a fairly neat usage of this - I would expect that if this entire thing is experimental this would be on the class itself but this is a decent way of getting the same thing. If this is how we wanna do this going forward it might be worth leaving a comment (both here and ExperimentalAttribute.cs)

Reply:
Yes I added the attribute there to show the warning at the top of the custom inspector 
![image](https://user-images.githubusercontent.com/53493796/73889569-db179600-4824-11ea-859a-49d27684db16.png)

Adding the attribute is in the documentation for an experimental feature: https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/ExperimentalFeatures.html#experimental-features-should-have-an-experimental-attribute